_posts/en/newsletters/2021-05-26-newsletter.md

```diff
@@ -0,0 +1,266 @@
+---
+title: 'Bitcoin Optech Newsletter #150'
+permalink: /en/newsletters/2021/05/26/
+name: 2021-05-26-newsletter
+slug: 2021-05-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a change of networks for several IRC
+channels and celebrates Optech's 150th newsletter.  Also included are
+our regular sections with popular questions and answers from the Bitcoin
+StackExchange, new software releases and release candidates, and notable
+changes to popular Bitcoin infrastructure projects.
+
+## News
+
+{% comment %}<!-- IRC move would probably be better as an Action Item,
+but I'd prefer not to have an empty News section if we can avoid it -harding -->{% endcomment %}
+
+- **IRC channels moving to Libera.Chat:** in the weekly Bitcoin Core
+  developer meeting, it was [decided][bccdev meeting libera] that the
+  meeting on Thursday, May 27th, will be the last meeting held on the
+  Freenode network.  Bots, logging, other infrastructure, future
+  meetings, and general discussion will be moved to `#bitcoin-core-dev`
+  on the [Libera.Chat][] network.  Several other channels related to
+  Bitcoin and LN are also moving.  For help finding the current network
+  for various channels, see the Bitcoin Wiki's [list of IRC channels][].
+  If you operate a channel that's moving and don't have a Wiki account
+  to update that list yourself, please let the editors know in
+  `#bitcoin-wiki` on Libera.
+
+## Celebrating Optech Newsletter #150
+*by [John Newbery][], Founder, Optech*
+
+This is the 150th regular Optech weekly newsletter that we've written for the
+Bitcoin technical community. Pausing only for short breaks around the Christmas
+holidays, we've published digests of the most important events in Bitcoin
+and Lightning development every week since June 2018.
+
+Optech was started with some very simple goals: to help Bitcoin businesses adopt
+technologies that allow Bitcoin to scale, and to highlight the amazing
+technical work happening in the open source Bitcoin community. Although we
+couldn't foresee exactly what form that would take three years ago, it's a
+mission that we continue to believe in, and that guides all the work we do.
+Since June 2018, we've:
+
+* Published 150 [newsletters][], numerous [blog posts][] and field reports,
+  a [special series on bech32][bech32], and an [interactive taproot workshop][]. In
```

Comment:
Maybe:
```suggestion
* Published 150 [newsletters][], numerous [blog posts][], field reports,
  a [special series on bech32][bech32], and an [interactive taproot workshop][]. In
```

Reply:
The 'numerous' refers to both 'blog posts' and 'field reports', so keeping this as one list item.